modules/relay/src/main/RelayApi.scala

```diff
@@ -137,9 +137,11 @@ final class RelayApi(
     val tour = data.make
     tourRepo.coll.insert.one(tour) inject tour
 
-  def tourUpdate(tour: RelayTour, data: RelayTourForm.Data)(using Me): Funit =
-    tourRepo.coll.update.one($id(tour.id), data.update(tour)).void andDo
-      leaderboard.invalidate(tour.id)
+  def tourUpdate(tour: RelayTour, data: RelayTourForm.Data)(using Me): Funit = for
+    subs <- tourRepo.subscribers(tour.id)
+    bson <- Future fromTry tourHandler.writeTry(data.update(tour))
+    _    <- tourRepo.coll.update.one($id(tour.id), bson ++ $doc("subscribers" -> subs))
+  yield leaderboard.invalidate(tour.id)
```

Comment:
not a fan of this solution, but at least it doesn't add as much code

Reply:
An alternative is just store subscriptions and notified in their own collection, but this seems more resilient as it prevents clobbering of stuff we add in the future. I think it runs too infrequently to worry about the number of db ops.

Reply:
If you go this route, you'll also want `notified`.

Reply:
dog damit

Reply:
I think the merged code will blow away spotlight and tier if a user with edit privs but no Relay permission submits the form?